pipelines/default_pipeline/demux_map/Snakefile

```diff
@@ -72,12 +72,13 @@ if config.get("minimum_identity"):
 else:
     minimum_identity= ""
 
+
 ##### Target rules #####
 
 rule all:
     input:
-        expand(config["output_path"]+ "/{filename_stem}.csv", filename_stem=config["filename_stem"])
-
+        csv = expand(config["output_path"]+ "/{filename_stem}.csv", filename_stem=config["filename_stem"]),
```

Comment:
These changes to default pipeline seem to me like we just forgot about them. We don't use the csv variable or do we? Same would apply to default_pipeline/demux_map/config.yaml

Reply:
no, we don't